app/models/movie.rb

```diff
@@ -21,40 +21,57 @@ class Movie < ApplicationRecord
   attr_accessor :production_companies
 
   scope :by_title, -> { order(:title) }
-  scope :by_shortest_runtime, -> { order(:runtime) }
-  scope :by_longest_runtime, -> { order(:runtime).reverse_order }
-  scope :by_recent_release_date, -> { order(:release_date).reverse_order }
-  scope :by_highest_vote_average, -> { order(:vote_average).reverse_order }
+  scope :by_shortest_runtime, -> { order(runtime: :asc) }
+  scope :by_longest_runtime, -> { order(runtime: :desc) }
+  scope :by_recent_release_date, -> { order(release_date: :desc) }
+  scope :by_highest_vote_average, -> { order(vote_average: :desc) }
   scope :default_list_order, -> (list) do
-    list.movies.order('CAST(listings.created_at as DATE) desc, listings.priority desc')
```

Comment:
This should fix that deprecation warning you mentioned, but I'm not sure how to test it.